src/Pages/Profile/admin/profile.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Badge } from 'reactstrap'
+const enums = require('../../../Enums.js')
+
+export function displayProfile (user, membershipValidUntil) {
```

Comment:
```suggestion
export function Profile (props) {
```
use props here. it's the convention for [functional components](https://programmingwithmosh.com/react/react-functional-components/)